apps/aecore/src/aec_peer_connection.erl

```diff
@@ -143,7 +143,7 @@ accept_init(Ref, TcpSock, ranch_tcp, Opts) ->
                 , {timeout, HSTimeout} ],
     %% Keep the socket passive until here to avoid premature message
     %% receiving...
-    inet:setopts(TcpSock, [{active, true}]),
+    inet:setopts(TcpSock, [{active, true}, {send_timeout, 1000}, {send_timeout_close, true}]),
```

Comment:
I see [send_timeout is related to send being completely accepted by underlying TCP stack and that it 
default timeout is infinity](https://github.com/erlang/otp/blob/OTP-20.2.3/lib/kernel/doc/src/inet.xml#L1057-L1063). I am concerned of 1s (this is ms, right?) hardcoded, that may be tight in real world scenarios. Would it make sense to make this [configurable](https://github.com/aeternity/epoch/blob/86772227ad547e05d0df21337b0f171e16ed258b/apps/aeutils/test/data/epoch_full.yaml#L29-L34)?

Reply:
I think 1s is a rather sensible default (after all we are talking about having micro-blocks in Bitcoin-NG with intervals in a handful(??) of seconds). Not sure it makes (much) sense to have it configurable it should at least always be significantly shorter than the default gen_server timeout or else there is a race!?